pkgs/development/python-modules/pycocotools/default.nix

```diff
@@ -0,0 +1,35 @@
+{ buildPythonPackage
+, cython
+, fetchPypi
+, lib
+, matplotlib
+, numpy
+, pytest
+}:
+buildPythonPackage rec {
+  pname = "pycocotools";
+  version = "2.0.0";
+
+  src = fetchPypi {
+    inherit pname version ;
+    sha256 = "cbb8c2fbab80450a67ee9879c63b0bc8a69e58dd9a0153d55de404c0d383a94b";
+  };
+
+  nativeBuildInputs = [
+    cython
+    numpy
+  ];
+
+  checkInputs = [
+    pytest
+    matplotlib
+  ];
+
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

Reply:
added `pythonImportsCheck`